### DISCUSSION TITLE
Support Italian language

### DISCUSSION THREAD

**Discussion by Angelk90** [community/community#16764] [Mobile]
Hi, would it be possible to add support for the Italian language?

**Comment by mxie** [comment]
Hi @Angelk90! We'd be happy to add support for more language if there's a demand for it but unfortunately this is not on our roadmap at the moment.

**Reply by Angelk90** [reply]
Hi @mxie , I understand, but if it published I don't know the db for example the json (if it is a json) of the words that need to be translated, I think you would find hundreds (thousands) of people ready to translate them, to also reduce the workload you have, too if I believe this thing is impossible.

**Reply by mxie** [reply]
It sounds like you're talking about translating repository content as opposed to the app text. Is that correct? If so, that is not something that we plan on offering.

**Reply by Angelk90** [reply]
@mxie: I'm not talking about the repository content, I'm talking about the Github mobile application.
